Automattic/wp-calypso
# Components: Refactor `TokenField` tests to RTL

#### Proposed Changes

This PR refactors the `TokenField` tests to use `@testing-library/react`.

These tests definitely need some more love, but the first step to improving them is to migrate them to use RTL. It's already a large PR, so I'd suggest that further improvements to tests should be in separate PRs.

This is the last item of #63409!

#### Testing Instructions

Verify tests still pass: `yarn run test-client client/components/token-field/test/index.jsx`

#### Pre-merge Checklist





Related to #63409
